internal/services/oracle/autonomous_database_regular_resource_test.go

```diff
@@ -0,0 +1,287 @@
+// Copyright © 2024, Oracle and/or its affiliates. All rights reserved
+
+package oracle_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/oracledatabase/2024-06-01/autonomousdatabases"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/oracle"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type AdbsRegularResource struct{}
+
+func (a AdbsRegularResource) Exists(ctx context.Context, client *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := autonomousdatabases.ParseAutonomousDatabaseID(state.ID)
+	if err != nil {
+		return nil, err
+	}
+	resp, err := client.Oracle.OracleClient.AutonomousDatabases.Get(ctx, *id)
+	if err != nil {
+		return nil, fmt.Errorf("retrieving adbs %s: %+v", id, err)
+	}
+	return pointer.To(resp.Model != nil), nil
+}
+
+func TestAdbsRegularResource_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, oracle.AutonomousDatabaseRegularResource{}.ResourceType(), "test")
+	r := AdbsRegularResource{}
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep(),
```

Comment:
The import step in the tests will fail since we can't retrieve this value from the API. Adding it as an argument here will tell the test to not expect it when we try importing the resource.
```suggestion
		data.ImportStep("admin_password"),
```
It will probably want to be added to all the tests

Reply:
Done.